numpy/core/include/numpy/ndarraytypes.h

```diff
@@ -1308,6 +1308,8 @@ typedef struct {
 #define PyArray_MultiIter_NOTDONE(multi)                \
         (_PyMIT(multi)->index < _PyMIT(multi)->size)
 
+#define PyArray_MultiIter_SIZE(multi)                \
+        _PyMIT(multi)->size
```

Comment:
If you really want to expose a new macro, it should be in a separate PR, not in a documentation PR.